config/config.go

```diff
@@ -27,6 +27,13 @@ const (
 )
 
 var windowsAbsolutePathPattern = regexp.MustCompile("^[A-Z]{1}:")
+// TODO: add jira issue tracker pattern
+var jiraOriginPattern = regexp.MustCompile(``)
+var githubOriginPattern = regexp.MustCompile(`^(https://)?(www\.)?github\.com/[a-zA-Z0-9\-_]+/[a-zA-Z0-9\-_]+`)
+var gitlabOriginPattern = regexp.MustCompile(`^(https://)?(www\.)?gitlab\.com/[a-zA-Z0-9\-_]+/[a-zA-Z0-9\-_]+`)
+var pivotalOriginPattern = regexp.MustCompile(`^(https://)?(www\.)?pivotaltracker/n/projects/[0-9]+`)
+// TODO: add redmine issue tracker pattern
+var redmineOriginPattern = regexp.MustCompile(``)
```

Comment:
For redmine, the regex should match any domain as it is self-hosted (e.g. `https://my-redmine-instance.com`)

There is no expected path appended to the hosts as redmine issues are accessed as `https://my-redmine-instance.com/issues/3`

Reply:
You can test integration \w Redmine by signing up to their demo hosted website - http://demo.redmine.org/

Reply:
resolved in accb53f